fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
No idea, I'm actually not that comfortable with my knowledge of Fedora once we get to Modeshape. Much less dealing with the resources directly inside. I would assume yes, I guess you'd start at the root and traverse every resource and check if you can HTTP GET them. If not then check the references in the JCR. Can you have a tool that does both?